test/e2e/accounts/common.ts

```diff
@@ -182,7 +183,20 @@ async function switchToAccount2(driver: Driver) {
 }
 
 export async function connectAccountToTestDapp(driver: Driver) {
```

Comment:
`switchToOrOpenDapp` attempts "an unusually fast switchToWindowWithTitle, just 1 second" and if that fails, then calls `openDapp`. The problem the `switchToWindowWithTitle` assumes that there will be a window other than the offscreen document present. It seems that `openDapp` fails if the drivers current window is the offscreen document. The change in this file ensures that, for this e2e test, the extension window is selected first and then the dapp can be opened from there.